keyform/templates/keyform/searchtable.html

```diff
@@ -0,0 +1,78 @@
+{% load i18n %}
+<div class="row">
+    <div class="large-12 columns">
+        <button id="search-button" class="button">{% trans 'Search' %}</button>
+        <br>
+    </div>
+</div>
+<div class="search-form" id="search">
```

Comment:
You should show this by default if `{{ data }}` is defined. Sidenote: maybe we should rename that to `search_data`.